doc/requirements.txt

```diff
@@ -20,6 +20,7 @@ docutils==0.20.1
     #   myst-parser
     #   sphinx
     #   sphinx-rtd-theme
+graphviz
```

Comment:
This actually needs to go [here](https://github.com/AIDASoft/podio/blob/master/doc/requirements.in) and then we use `pip-compile` to generate the actual `requirements.txt` from there.

Reply:
@tmadlener Thanks for explaining, I've updated `requirements.in` . I'm sorry for adding the dependency to an auto-generated file 😅